### PR TITLE
Update the ORAS website with new design and Docusaurus framework

### PR DESCRIPTION
**Pull request by FeynmanZhou**
Fix https://github.com/oras-project/oras-www/issues/82 as the first phase of the new ORAS website is completed by @asmitbm . 
The go module redirect configuration is resolved in #146, see https://github.com/oras-project/oras-www/pull/146#issuecomment-1522087967. 
You can preview the new website [here](https://deploy-preview-147--oras-project.netlify.app/).

The next phase plan is listed in https://docs.google.com/document/d/1C8imk1kocTRxAYWMi9u20ZPejUNySIBWXH1xqF6b9rk/edit?pli=1

Preview: https://deploy-preview-147--oras-project.netlify.app/

### CODE REVIEW

**Review by SteveLasker — APPROVED** [review]
Looks great, thank you @FeynmanZhou and others that have contributed.
LGTM for the overall design, with content tweaks coming later.
It's also approved with https://github.com/oras-project/oras-www/pull/146 merged to assure we're not breaking the go module redirect issue.

**Review by sabre1041 — APPROVED** [review]
LGTM

File: docs/CLI_Reference/use_oras_cli.mdx
Comment (TerryHowe) [inline docs/CLI_Reference/use_oras_cli.mdx:59]:
```suggestion
```

I think in the current version of the docs I removed all this since it repeats what is in the menu and we have to maintain it. Not a blocker right now though

**Review by TerryHowe — APPROVED** [review]
LGTM

File: docs/CLI_Reference/use_oras_cli.mdx
Comment (FeynmanZhou) [reply docs/CLI_Reference/use_oras_cli.mdx:59]: Agree. We could remove these content in a follow-up PR

**Review by shizhMSFT — APPROVED** [review]
LGTM

**Comment by FeynmanZhou** [comment]
We got the majority of approval from the ORAS maintainers. It's ready to merge this PR.

**Pull request merged**
